### PR TITLE
feat: stop web worker & prefetch and loading script via url parameter & add lint rules

### PR DESCRIPTION
**Pull request by akitaSummer**
- [x] stop web worker
- [x] prefetch and loading script via url parameter
- [x] add lint rules

local preview:
http://localhost:3000/?auto_run=true&your_url=https%3A%2F%2Fgist.githubusercontent.com%2Flidangzzz%2F2adebd6a771f50e6430a34ec61d928af%2Fraw%2F1f9add39370b00dfbb3f5d20c8bc203dd09f52c2%2Fmy_first_notebook.hss

### CODE REVIEW

**Comment by AGDholo** [comment]
Thank you for your contribution. This is a great PR.
I have a little doubt, should we make it part of `hedgehog-lab core`?

cc @imtsuki @lidangzzz

**Comment by AGDholo** [comment]
Online preview: https://hedgehog-lab-git-feature-akita.sniper.vercel.app/
Auto run: https://hedgehog-lab-git-feature-akita.sniper.vercel.app/?auto_run=true&your_url=https%3A%2F%2Fgist.githubusercontent.com%2Flidangzzz%2F2adebd6a771f50e6430a34ec61d928af%2Fraw%2F1f9add39370b00dfbb3f5d20c8bc203dd09f52c2%2Fmy_first_notebook.hss

**Comment by promer94** [comment]
1. I suggest it would be better if we could make use of React Query. React Query exposes a generic way to cancel a Promise-like query. You can see the example that i made at #76.
2. I prefer [ky](https://github.com/sindresorhus/ky) than axios because ky is more typescript friendly than axios.

File: packages/hedgehog-lab/src/hooks/index.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const useQuery = () => {
```
Comment (promer94) [inline packages/hedgehog-lab/src/hooks/index.ts:1]: I recommend to use [qs](https://github.com/ljharb/qs) here

**Comment by akitaSummer** [comment]
@promer94 Thank you for your suggestion.I have merged your branch from your github and replaced axios.

**Comment by AGDholo** [comment]
Thank you very much for your contribution

Should we need to delete the #76 ?
Is this code review resolved? 
Is the branch ready to merge?

Should this change be on `hedgehog-core` or `hedgehog-lab`? It looks like a front-end web enhancement, and I prefer to move it to `hedgehog-lab`.

**Comment by promer94** [comment]
i will close #76

**Comment by akitaSummer** [comment]
I think it can be merged into the master. I modified the `hedgehog-lab` like `hedgehog-core` because it was added prettier for unified code style.If you don't like it, you can only merge `hedgehog-lab`.

**Comment by AGDholo** [comment]
Merged now, thanks!

**Pull request merged**